interface/forms/LBF/printable.php

```diff
@@ -97,10 +97,10 @@
         'default_font' => '',
         'margin_left' => $GLOBALS['pdf_left_margin'],
         'margin_right' => $GLOBALS['pdf_right_margin'],
-        'margin_top' => $GLOBALS['pdf_top_margin'],
-        'margin_bottom' => $GLOBALS['pdf_bottom_margin'],
-        'margin_header' => '',
-        'margin_footer' => '',
+        'margin_top' => $GLOBALS['pdf_top_margin']*2,
+        'margin_bottom' => $GLOBALS['pdf_bottom_margin']*2,
```

Comment:
note had to multiple this by 2 so the text in the header/footer did not overlap with the main text